_pages/team.md

```diff
@@ -1,16 +1,58 @@
+
+## &#10070;
+
+<img src="https://projectvikram.github.io/restricted_assets/images/dr_srija.jpg" alttext="Dr Srija Katta (avatar image)" width="300" height="300">
+
+## Dr Srija Katta 
+### Technology Evangelist-Pharma and Convenor
+#### @srija-ieee
+
+Dr Srija is the Co-Founder of AyeAI Consulting and Heads Company Operations and Pharma lab. She is a Director at Vyas Labs Pvt Ltd and Interglial Healthcare (OPC) Pvt Ltd. 
+
+She is a passionate Entrepreneur experienced in Startup strategy, New Product Road Map, Requirement Analysis, with skills in Artificial Intelligence for Pharma and Healthcare. She was one among the invitees of Roundtable “Transforming Primary Healthcare – Leveraging Technologies” at NASSCOM Technology and Leadership Forum 2019. She was an invited speaker for IEEE CIS Summer School 2018 and has delivered a handson session on topic “AI in Healthcare” Dr Srija is familiar with Course design, Seminars and Workshop delivery. 
+
+Dr Srija is a Clinical Pharmacist (Doctor of Pharmacy) by profession, trained in Biz ops like Regulatory Compliance, Banking Operations, HR management, Public Relations, Business development Collaterals design, Awareness and Marketing Campaign design and other technical aspects like Gitops, Website building etc.. She is skilled in Systemic and Conversational Peer Review and has been a Reviewer for a few international conferences and journals including International Society of Pharmacoeconomics and Outcomes Research 6 TH Latin America Conference, Brazil - 2017 and 20th Annual European Congress.-2017.
 
-## Dr Srija Katta
-### Co-Convener
+Her professional Interest includes Artificial Intelligence and Insilico based accelerated drug discovery, Medical devices, Digihealth, Telemedicine, IoMT. Research interests include Psychoneuroimmunological studies with respect to human epigenetics and social habits. She aspires for Social Welfare and has conducted various Healthcare Awareness Campaign during her graduation days. She actively coordinates the activities of TWISHA - The Women in Self Health Awareness, a health4all initiative by an NGO, Gramsheel Foundation. She has been one of among the top distinction holder of the prestigious Ashoka Changemaker Intraprenuership - 2018
```

Comment:
Interest -> interests
Insilico in capitals is a brand name or say Insilico Drug Discovery
Insilico means "silicon based"
D capital in Medical Devices
an NGO, -> no "an NGO,"
"one of among" -> one among
holder -> holders

Reply:
dint get the change in 'an NGO'